save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -249,23 +249,25 @@ class VulnerabilityService(
     }
 
     /**
-     * @param vulnerabilityDtos dto list of vulnerability projects
+     * @param vulnerabilityProjectDtos dto list of vulnerability projects
+     * @return [VulnerabilityDto] with saved [vulnerabilityProjectDtos]
      */
     @Transactional
-    fun saveAllProjects(vulnerabilityDtos: List<VulnerabilityProjectDto>) {
-        vulnerabilityRepository.findByName(vulnerabilityDtos.first().vulnerabilityName)?.let { vulnerability ->
-            vulnerabilityProjectRepository.saveAll(
-                vulnerabilityDtos.map { dto ->
-                    VulnerabilityProject(
-                        name = dto.name,
-                        url = dto.url,
-                        versions = dto.versions,
-                        type = dto.type,
-                        vulnerability = vulnerability,
-                    )
-                }
-            )
-        }
+    fun saveAllProjects(
+        vulnerabilityProjectDtos: List<VulnerabilityProjectDto>
+    ): List<VulnerabilityProjectDto>? = vulnerabilityRepository.findByName(vulnerabilityProjectDtos.first().vulnerabilityName)?.let { vulnerability ->
```

Comment:
you changed the return type, but it doesn't lead to change anywhere. Do we need this return value?

Reply:
It will be used it @Cheshiriks's PR